packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md

```diff
@@ -13,6 +13,8 @@ Start by signing in to the Okta Developer Console:
 
 3. Next, enter values for the **Login redirect URI**. This is the callback from <GuideLink link="../define-callback/">Define a callback route</GuideLink>. Add values for local development (such as `http://localhost:8080/login/callback`) and production (such as `https://app.example.com/login/callback`).
 
+    > **Note:** When you add an endpoint as a **Login redirect URI**, it is automatically added to the CORS allowed list.
```

Comment:
This isn't true- all that happens is that when you're redeeming your code for a token we add CORS headers if the request origin is in your redirect uris

Reply:
You would be able to exchange your code for access and ID Tokens from your SPA application as long as the request origin is configured as a redirect URIs. In that case, Okta will automatically add CORS headers to the request.

Reply:
How about this? ^^